CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [3.6.0](https://github.com/marklogic-community/mlxprs/releases/tag/v3.6.0)
+
+- [#116](https://github.com/marklogic-community/mlxprs/issues/116) Updated dependencies and fixed all known security vulnerabilities
+- [#119](https://github.com/marklogic-community/mlxprs/issues/119) Added SJS binding to javascript language
+- [#97](https://github.com/marklogic-community/mlxprs/issues/97) Added a configuration option for the MarkLogic Manage AppServer port (used for the SJS debug server port)
+- [#117](https://github.com/marklogic-community/mlxprs/issues/117) Fixed a bug with the first attempt to eval a JavaScript tab
+- [#109](https://github.com/marklogic-community/mlxprs/issues/109) Prevent the database param from being used in REST calls when the database setting is empty
+- Many updates to the project to facility future enhancements and maintenance
```

Comment:
Release notes should only be for users, so remove this section since it's intended for developers. Makes it easy for users to scan release notes to find items of interest.